src/MCPClient/tests/test_antivirus.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf8 -*-
+"""Tests for the archivematicaClamscan.py client script."""
+
+import os
+import sys
+
+from collections import OrderedDict, namedtuple
+
+import pytest
+import test_clamdscan
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(
+    os.path.abspath(os.path.join(THIS_DIR, '../lib/clientScripts')))
+
+import archivematicaClamscan
+
+# Antivirus Tests
```

Comment:
This sounds like documentation at the module level, right? If that's the case, you already have a docstring for the module in L2 that you may want to update instead. I don't know if that was me or you who wrote it but worth fixing it either way :)

Reply:
Resolved.